core/src/com/mygdx/insanerpg/model/unit/Movable.java

```diff
@@ -0,0 +1,5 @@
+package com.mygdx.insanerpg.model.unit;
+
+public interface Movable {
```

Comment:
Might want to have another declaration of void Move() but with no args for mobs moving

Reply:
added a void Move() constructor